core/domain/html_cleaner.py

```diff
@@ -128,7 +136,12 @@ def strip_html_tags(html_string: str) -> str:
         str. The HTML string that results after all the tags and attributes are
         stripped out.
     """
-    return bleach.clean(html_string, tags=[], attributes={}, strip=True)
+    # Here we use MyPy ignore because bleach.clean's attributes parameter
```

Comment:
Should we then use attributes=None? What does bleach expect (can you link to the docs)?